frame/clearing-house/src/tests/close_position.rs

```diff
@@ -248,7 +251,7 @@ fn should_fail_if_pushes_index_mark_divergence_above_threshold() {
 		));
 
 		// Alice tries to close her position, but it fails because it pushes the mark price too
-		// below the index Closing tanks mark to 89% of previous price
+		// below the index. Closing tanks the mark price to 89% of previous one.
```

Comment:
"Closing tanks the mark price to 89% of *the* previous one." ? (Not sure about it though, I'm not good at grammar :smiling_face_with_tear: )

Reply:
You are right ser